pkg/option/config.go

```diff
@@ -3125,6 +3146,7 @@ func (c *DaemonConfig) Populate() {
 	c.HTTP403Message = viper.GetString(HTTP403Message)
 	c.DisableEnvoyVersionCheck = viper.GetBool(DisableEnvoyVersionCheck)
 	c.K8sNamespace = viper.GetString(K8sNamespaceName)
+	c.AgentNotReadyNodeTaintKey = viper.GetString(AgentNotReadyNodeTaintKeyName)
```

Comment:
We are missing a corresponding `flag.String(option.AgentNotReadyNodeTaintKeyName, …)` setup (I'm guessing in `daemon/cmd/daemon_main.go` `initializeFlags()`) correct?

Reply:
Added. Thanks for the review.